sdk/src/main/java/io/dapr/actors/DaprAsyncClient.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ */
+
+package io.dapr.actors;
+
+import reactor.core.publisher.Mono;
+
+/**
+ * Interface for interacting with Dapr runtime.
+ */
+public interface DaprAsyncClient {

```

Comment:
Doesn't need to be public, this will used by other public Apis in SDK. eg. ActorProxy on client side & ActorRuntime classes

Reply:
For now, made it visible to package. If it needs to be accessed by another Java package (not jar) it will probably need to be made public again.